src/frontends/onnx/frontend/src/op/softmax_cross_entropy_loss.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/deprecated.hpp"
+OPENVINO_SUPPRESS_DEPRECATED_START
+
+#include "ngraph/node.hpp"
+#include "onnx_import/core/node.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+
+OutputVector softmax_cross_entropy_loss(const Node& node);
```

Comment:
Please sync with the latest master. ONNX FE has been slightly refactored. The changes (namespaces, headers) need to be aligned with the rest of the frontends/onnx files.

Any of the onnx frontend op file can be taken as an example to follow, like:

[/src/frontends/onnx/frontend/src/op/leaky_relu.hpp](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/leaky_relu.hpp)
[/src/frontends/onnx/frontend/src/op/leaky_relu.cpp](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/leaky_relu.cpp)